sdk/remoterendering/azure-mixedreality-remoterendering/src/test/java/com/azure/mixedreality/remoterendering/RemoteRenderingClientTest.java

```diff
@@ -138,7 +139,7 @@ public void sessionTest(HttpClient httpClient) {
         RemoteRenderingClient client = getClient(httpClient);
         BeginSessionOptions options = new BeginSessionOptions().setMaxLeaseTime(Duration.ofMinutes(4)).setSize(RenderingSessionSize.STANDARD);
 
-        String sessionId = getRandomId("sessionTest");
+        String sessionId = getRandomId("sessionTest2");
```

Comment:
Why change this string (same with asyncSessionTest to asyncSessionTest2)?

Was it just to distinguish the recorded tests?

Reply:
Ah yes, it was because I already had a session with an older name (sessionTest), and I couldn't find a way to delete it in the portal, so for re-recording purposes I used a new name as the prefix.